lib/DPNSClient.js

```diff
@@ -7,21 +7,32 @@ const resolveMethodFactory = require('./method/resolveMethodFactory');
 const resolveByRecordMethodFactory = require('./method/resolveByRecordMethodFactory');
 const searchMethodFactory = require('./method/searchMethodFactory');
 
+const InvalidArgumentError = require('./errors/InvalidArgumentError');
+
 /**
  * DPNS client
  */
 class DPNSClient {
   /**
    * @param {DAPIClient} dapiClient
    * @param {Wallet} wallet
+   * @param {string} dpnsIdentityId
    */
-  constructor(dapiClient, wallet) {
+  constructor(dapiClient, wallet, dpnsIdentityId) {
     const dpp = new DashPlatformProtocol({
       dataProvider: {},
     });
 
+    const isEnvPresent = process && process.env;
```

Comment:
I think we should remove `process.env` from the client code and pass `process.env.DPNS_IDENTITY_ID` in tests as the third argument.